### PR TITLE
update with latest on-brand specs

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

Implements latest brand direction for Product and Web theme experiences.

#### What are the relevant issues?

https://github.com/grommet/hpe-design-system/issues/3102

### CODE REVIEW

File: src/js/themes/hpe.js [hunk 1]
```diff
@@ -338,50 +378,18 @@ export const hpe = deepFreeze({
       },
     },
     disabled: {
```
Comment (taysea) [inline src/js/themes/hpe.js:380]: I actually think we can remove this entire block of "disabled". `opacity: 0.3` comes from grommet theme (we can leave here to be explicit if we want) and the `undefined` overrides for button kinds/background/etc. were only there to override what we had in grommet-theme-hpe (which will now be removed).

**Review by taysea — APPROVED** [review]
One small comment about `button.disabled` cleanups but it doesn't have to block merging this PR. we can cleanup later.

File: src/js/themes/hpe.js (hunk 1 above)
Comment (halocline) [reply src/js/themes/hpe.js]: done.

**Review by taysea — APPROVED** [review]
LGTM. Thanks for transferring it over here.

**Pull request merged**